frontends/ol-components/src/components/SearchInput/SearchInput.tsx

```diff
@@ -15,6 +18,29 @@ export interface SearchSubmissionEvent {
   preventDefault: () => void
 }
 
+const StyledInput = styled(Input)`
+  border-top-right-radius: 0;
+  border-bottom-right-radius: 0;
+  max-width: 81%;
+  border-right: none;
+  height: 37px;
```

Comment:
`theme.typography.body2` here. The designs show the placeholder with color `silverGrayDark`. Not sure about the active text, probably `darkGray2` again.

On mobile it's `theme.typography.body4`

Reply:
Also adds `padding: 8px 16px;`

<img width="599" alt="image" src="https://github.com/mitodl/mit-open/assets/939376/f84fa7bc-e2ad-4c6d-b475-d862547509de">